### ISSUE TITLE
Hide event is called twice when color picker is closed

### ISSUE THREAD

**Issue by kyle-jennings** [claviska/jquery-minicolors#230]
When clicking off of the color picker, the hide callback is called twice.  This can be reproduced by cloning the repo and adding the following code to index.html after line 88:
```
hide: function(){
  console.log('hide');
},
show: function(){
  console.log('show');
},
```

Now if you reload the demo and open your browser console, you'll see that when the color picker opens "show" is echoed a single time.  But when you click off the color picker to close it, "hide" is displayed twice.

I just discovered the issue and haven't had a chance to troubleshoot.

**Comment by kyle-jennings** [comment]
@claviska I've tracked down the culprit:

https://github.com/claviska/jquery-minicolors/blob/master/jquery.minicolors.js#L978
Also an issue in #226 

I think it can be fixed but changing 
`$([document, top.document])` to `$( top.document).`
I'll make a a PR with this fix.

**Comment by claviska** [comment]
This bug was introduced in ea7ed2bbfdffa0a6045f719830bed9bc5dea2a3d. I'm OK with reverting this behavior to fix the bug, but we may want to introduce a new option that lets users specify which document to listen on. See #206.

/cc @andreasfernandez

**Comment by claviska** [comment]
Fixed in #231
